.travis.yml

```diff
@@ -81,7 +81,10 @@ jobs:
     - stage: Test
       install: skip
       env: DB=POSTGRES_10
-      script: ci/test.sh "$DB" && export JAVA_HOME=/usr/local/lib/jvm/openjdk8 && ci/test.sh WILDFLY
+      script: ci/test.sh "$DB"
+        && ./mvnw -q install -f history -DskipTests -Dmaven.javadoc.skip -Dcheckstyle.skip -Dasciidoctor.skip
```

Comment:
We are only installing the history module for the wildfly tests. That's why ;)

Reply:
The wildfly need the history module. Whenever we introduced that we didn't adapt our build process to it.

Reply:
I realized this because I introduced breaking changes and the build did not pull those changes. - I figured out that wildfly  uses the history module from sonatype and not from the current build.